ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import logging
+from asyncio import run
+from json import loads
+
+from ansible.plugins.connection import ConnectionBase
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
```

Comment:
please use relative imports

Reply:
This is using the same import as `eos_designs_shared_utils` and `eos_designs_facts` plugins - should we change all the plugins imports?
https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/utils.py#L9
https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts/eos_designs_facts.py#L9